scripts/run-scss-lint.sh

```diff
@@ -5,5 +5,5 @@
 PROJECT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd)
 
 docker run --rm \
-  -v $PROJECT_DIR/src:/style-guide/src \
-  brainly/style-guide scss-lint
+  -v $PROJECT_DIR:/style-guide/ \
+  brainly/style-guide scss-lint $1
```

Comment:
Minor: instead `$1` you could pass parameters using `"$@"` (`"`'s are important) that would just proxy all given CLI parameters to container's entrypoint.